doc/source/text.rst

```diff
@@ -429,6 +429,27 @@ String ``Index`` also supports ``get_dummies`` which returns a ``MultiIndex``.
 
 See also :func:`~pandas.get_dummies`.
 
+Concatenating Series
+--------------------
+
+The method :meth:`Series.str.cat` can be used to concatenate the records of two :class:`Series`. Depending on the value given to the ``align`` keyword,
+their indexes will be aligned before concatenation (if ``align=True``) or not (if ``align=False``). As usual, alignment will expand to the union of both
+indexes, while introducing ``NaN`` for missing values in the respective other series (which can be easily handled with the ``na_rep``-keyword).
+
+If the ``align`` keyword is not passed, the method will currently fall back to the previous behavior (i.e. ``align=False``),
+but a ``FutureWarning`` will be raised, since this default will change to ``align=True`` in a future version of ``pandas``.
+
+.. ipython:: python
+
+    base = Series(['a', 'b', 'c', 'd', 'e'])
+	s = base.reindex([0, 1, 2, 3])
+	t = base.reindex([3, 0, 4, 1])
+    s.str.cat(t, align=False)
+	s.str.cat(t, align=True)
+	s.str.cat(t, align=False, na_rep='')
+
+.. versionadded:: 0.23.0
```

Comment:
It's unclear what this `versionadded` is referring to.  The keyword? The `.str.cat` method? I think it's best to leave that to the API documentation.

Reply:
Fixed